Orange/data/variable.py

```diff
@@ -232,51 +32,47 @@ def __new__(cls, name, bases, attrs):
         return obj
 
 
-class Variable(metaclass=VariableMeta):
+class Variable(str, metaclass=VariableMeta):
     """
     The base class for variable descriptors contains the variable's
-    name and some basic properties.
-
-    .. attribute:: name
+    name and some basic properties. This extends str so it plays nicely with
+    pandas' column values.
 
+    Attributes
+    ----------
+    name : str
         The name of the variable.
-
-    .. attribute:: unknown_str
-
+    unknown_str : set
         A set of values that represent unknowns in conversion from textual
-        formats. Default is `{"?", ".", "", "NA", "~", None}`.
-
-    .. attribute:: compute_value
-
+        formats. Default is Variable.MISSING_VALUES.
```

Comment:
So why would this be defined twice? Just have `Variable.MISSING_VALUES` and that's it.